### DISCUSSION TITLE
Section 1: Cannot install Foundry-zksync on windows Ubuntu

### DISCUSSION THREAD

**Discussion by Vincent71399** [Cyfrin/foundry-full-course-cu#2085] [Q&A]
after I download foundry-zksync git repo, in root directory, if I type in: ./install-foundry-zksync
I will get error
-bash: ./install-foundry-zksync: /bin/bash^M: bad interpreter: No such file or directory
but the file install-foundry-zksync is in the current directory

I tried what chatgpt suggested: dos2unix install-foundry-zksync
then after type in ./install-foundry-zksync I got another error:

Navigating to the foundryup-zksync script...
Making install script executable...
Running the installation script...
/usr/bin/env: ‘bash\r’: No such file or directory
No shell configuration file detected. Please source your shell manually or start a new terminal session.
Running foundryup-zksync setup...
/usr/bin/env: ‘bash\r’: No such file or directory

anyone knows how to resolve it?

**Comment by EngrPips** [comment]
I am unsure what could be wrong, but let's wait to hear from @cromewar  when they are available.

**Comment by Okiki-omisande** [comment]
**Hi, I encountered the same issue but was able to resolve it with the following steps:**

1. **Navigate to the Cloned Repository Folder:**
   - I went into the folder where I had cloned the `matter-lab` repository.

2. **List the Contents:**
   - I ran the `ls` command to list the contents of the current directory.

3. **Locate the `foundry-zksync` Folder:**
   - In the list, I saw a folder named `foundry-zksync`.

4. **Change Directory to `foundry-zksync`:**
   - I navigated into the `foundry-zksync` folder using the following command:
     ```bash
     cd foundry-zksync
     ```

5. **Run the Installation Script:**
   - Once inside the `foundry-zksync` folder, I executed the installation script with:
     ```bash
     ./install-foundry-zksync
     ```

**And that worked for me.**

**Reply by EngrPips** [reply]
Hello @Vincent71399, Please can you follow @Okiki-omisande suggestion, and let's see if it helps you solve your issue.
